### DISCUSSION TITLE
Update the value of scale

### DISCUSSION THREAD

**Discussion by Nicolerobinn** [pmndrs/use-gesture#466] [Q&A] [answered]
When I update the scale from the outside and use onPinch to scale again, I get the wrong value for the scale

**Comment by dbismut** [comment]
Wow how am I even supposed to help with that little context?

**Reply by Nicolerobinn** [reply]
I don't know how to describe it, because it's a hands-on problem, right

**Reply by Nicolerobinn** [reply]
This is the problem that exists in the example of your official website

**Reply by dbismut** [reply]
Just try. I'll delete in a couple of days if this remains as vague as it is now.

**Reply by Nicolerobinn** [reply]
[This is the example I just changed，After entering, reduce the image to the minimum，and then click on the button ，Finally, gently shrink the image，You'll see that the image shrinks to a minimum in an instant](https://codesandbox.io/live/8697892149d
)

**Comment by dbismut** [comment, marked as answer]
Would be nice if you could tidy up comments next time instead of spamming the thread.
Here's what you're looking for: https://codesandbox.io/live/8697892149d?file=/src/App.jsx

Related documentation: https://use-gesture.netlify.app/docs/options/#from

**Reply by Nicolerobinn** [reply]
I found a solution.
